psp-examples/swap/circuits/swaps/swaps.circom

```diff
@@ -0,0 +1,335 @@
+/**
```

Comment:
This file is unrelated to the goal of this PR, but I think it's a good change to make? Seems like this circuit was never commited, even though it's clearly produced by macro-circum whenever the swap PSP is built.